components/Common/Breadcrumbs/BreadcrumbLink/index.module.css

```diff
@@ -1,4 +1,7 @@
 .link {
+  @apply w-full 
+  truncate;
```

Comment:
```suggestion
    truncate;
```

I forgot to mention that we also add extra spaces for items from following lines (as you can see on the other lines hehe)

Reply:
haha will commit in a minute

Reply:
I already did it 🫡